lib/interface/cli/commands/workflow/wait.cmd.js

```diff
@@ -26,9 +26,16 @@ const annotate = new Command({
                 default: 'success',
                 required: true,
             })
+            .option('verbose', {
```

Comment:
I have added it because --debug option is not boolean, that is a bit confusing bcs you need to pass some random string next to the option. Also, we can't just change the type because there is a chance to break some customers' automation. So the solution to exclude it from the docs and add another similar option that was done right.